magento/magento2
# 24978 fixed popup design in new email template adminhtml

### Description (*)
The js confirmation popup design is not proper in Add/Edit Email template at adminhtml. Please follow below steps to reproduce the issue.


### Fixed Issues (if relevant)

1. magento/magento2#24978: New Email Template JS Confirm Message Popup Design is not Proper

### Manual testing scenarios (*)

1.Login Admin -> Marketing
2. Under Communication Section -> Open Email Templates and Newsletter Templates both in different browser tab.
3. Click on Add New Template button on both the views.
4. Click on Convert to Plain Text button.
5. Observe the js Confirmation Popup of both the views is different.

### Questions or comments


### Contribution checklist (*)
